pipelines/common/src/main/java/com/google/fhir/analytics/ParquetUtil.java

```diff
@@ -324,17 +324,22 @@ public void writeRecords(Bundle bundle, Set<String> resourceTypes)
       if (resourceTypes == null || resourceTypes.contains(resource.getResourceType().name())) {
         write(resource);
         if (createParquetViews) {
-          ImmutableList<ViewDefinition> views = viewManager.getViewsForType(resource.fhirType());
-          if (views != null) {
-            for (ViewDefinition vDef : views) {
-              write(resource, vDef);
-            }
-          }
+          writeViews(resource);
         }
       }
     }
   }
 
+  public void writeViews(Resource resource)
```

Comment:
Instead of doing this, I think we can move this functionality in `write(resource)` method above, correct? I mean, I prefer that we do not expose `writeViews` as a separate method in the public API of this class. Instead, since we have `createParquetViews` already, decide inside `ParquetUtil` when to write views and when not.

Reply:
Done.